ios/RCTTWVideoModule.m

```diff
@@ -206,10 +207,17 @@ - (void)addParticipantView:(TVIVideoView *)view sid:(NSString *)sid trackSid:(NS
   if (self.localVideoTrack != nil) {
     [self.localVideoTrack setEnabled:enabled];
     resolve(@(enabled));
-  } else {
-    [self.localVideoTrack setEnabled:false];
-    resolve(@false);
   }
+  else{
+    [self createLocalVideoTrack];
```

Comment:
If think we should only create this track if enabled is true.

Reply:
I have corrected it.